common/config/rush/browser-approved-packages.json

```diff
@@ -1,5 +1,10 @@
 // DO NOT ADD COMMENTS IN THIS FILE.  They will be lost when the Rush tool resaves it.
 {
   "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/approved-packages.schema.json",
-  "packages": []
+  "packages": [
+    {
+      "name": "jszip",
+      "allowedCategories": [ "libraries" ]
+    }
```

Comment:
This entry should go in `nonbrowser-approved-packages.json`

Reply:
FYI we researched several zipping packages:
- https://www.npmjs.com/package/bestzip - relies on an external native binary
- https://www.npmjs.com/package/adm-zip - has zero dependencies, but is less widely used than jszip
- https://www.npmjs.com/package/jszip - seemed to have good docs/support and reasonable dependenices; relies on `pako`  for compression boasts various optimizations
- https://www.npmjs.com/package/archiver - brings along quite a lot of dependencies. Also I didn't like its multi-format abstraction, since in my experience file formats have important special considerations (e.g. zip can add new items, tar has chmod bits, etc) that are maybe more awkward behind an abstraction